src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -526,7 +525,9 @@ def __update_cluster_status(self):
             )
 
             self.dal.publish_cluster_status(cluster_status)
-            self.dal.publish_cluster_nodes(cluster_stats.nodes, self.__pods_running_count)
+            avg_cpu = self.__prometheus_discovery_util.get_cluster_avg_cpu()
```

Comment:
I would put this in `try/except`, so if we have an error querying prometheus, we still publish the cluster nodes